django_zero_downtime_migrations/backends/postgres/schema.py

```diff
@@ -143,7 +144,7 @@ class PGShareUpdateExclusive(PGLock):
     pass
 
 
-class DatabaseSchemaEditorMixin:
+class DatabaseSchemaEditorMixin(BaseDatabaseSchemaEditor):
```

Comment:
Will need to revert this... Added for better "go to definition" support in my editor, forgot to remove before pushing